server/services/courtRegisterService.test.ts

```diff
@@ -79,6 +79,54 @@ describe('Court Register service', () => {
 
       expect(court.courtId).toEqual('SHFCC')
     })
+    it('will throw error when not found', async () => {
+      fakeCourtRegister.get('/courts/id/SHFCC').reply(404, {
+        status: 404,
+        developerMessage: 'Court SHFCC not found',
+      })
+
+      try {
+        await courtRegisterService.getCourt({}, 'SHFCC')
+        expect(true).toBe(false)
```

Comment:
I guess this should never happen - is there a way to add some text to this to indicate that in the failing test?

Reply:
Hopefully, I will take a look